torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -162,17 +175,33 @@ struct GraphFuser {
     if (node->matches("aten::add(Tensor self, Tensor other, *, Scalar alpha) -> Tensor", /*const=*/attr::alpha)) {
       std::vector<Value*> inputs {node->namedInput(attr::self), node->namedInput(attr::other)};
       return areTensorsOfSameShape(inputs) && haveSupportedType(inputs);
-    } else if (node->matches("aten::lt(Tensor self, Tensor other) -> Tensor") ||
-               node->matches("aten::le(Tensor self, Tensor other) -> Tensor") ||
-               node->matches("aten::gt(Tensor self, Tensor other) -> Tensor") ||
-               node->matches("aten::ge(Tensor self, Tensor other) -> Tensor") ||
-               node->matches("aten::eq(Tensor self, Tensor other) -> Tensor") ||
-               node->matches("aten::ne(Tensor self, Tensor other) -> Tensor")) {
+    } else if (isComparisonOp(node)) {
       // comparison operators produce Byte type, and it's ok, check only inputs
       return areTensorsOfSameShape(node->inputs()) && haveSupportedType(node->inputs());
     } else if (node->matches("aten::type_as(Tensor self, Tensor other) -> Tensor")) {
       // type_as can have different input types as long as output is float, check only output
       return haveSupportedType(node->outputs());
+    } else {
+      return hasSupportedShapeAndType(node);
+    }
+  }
+
+  // Can the operator be fusable after adding adding explicit expands to inputs?
+  // XXX: Somehow de-duplicate this and isFusable.
```

Comment:
This isn't maintainable until isFusible is merged with this. Already it is the case that it is being put into conditions right next to isFusible.